src/cpp/fastdds/publisher/DataWriterImpl.hpp

```diff
@@ -188,6 +188,14 @@ class DataWriterImpl
     //! Remove all listeners in the hierarchy to allow a quiet destruction
     void disable();
 
+    /**
+     * Removes all changes from the History.
+     * @param[out] removed Number of removed elements
+     * @return True if correct.
+     */
+    bool remove_all_changes(
```

Comment:
Change if the method's name finally changes to `clear_history` as proposed elsewhere